### PR TITLE
Zastrow style updates

### PR DESCRIPTION
**Pull request by zastrow**

### CODE REVIEW

File: src/scss/_components.diagram.chart.scss [hunk 1]
```diff
@@ -1,5 +1,24 @@
 $circle-height: 28px;
 
+@mixin chart-bar($color) {
```
Comment (mdt2) [inline src/scss/_components.diagram.chart.scss]: Nice refactor 🎉

File: src/scss/_components.diagram.chart.scss [hunk 2]
```diff
@@ -165,117 +184,35 @@ $circle-height: 28px;
...
-        border: solid 2px $white;
-      }
+      @include chart-bar($white);
+    }
 
-      &::after {
-        box-shadow: 0 0.125rem 1rem $white;
-      }
+    &--bluegreen {
+      @include chart-bar(mix($blue, $green, 50%));
```
Comment (mdt2) [inline src/scss/_components.diagram.chart.scss]: Should these `bluegreen` and `yellowpink` mixes be color variables?

File: src/scss/_components.skip-link.scss [hunk 3]
```diff
@@ -2,7 +2,7 @@
   @include body-copy;
   font-weight: 700;
   display: block;
-  left: 0;
+  right: 100%;
```
Comment (mdt2) [inline src/scss/_components.skip-link.scss]: This change is pushing the skip link off the page when you tab to it
 
skip-link

File: src/scss/_elements.layout.scss [hunk 4]
```diff
@@ -1,6 +1,33 @@
...
+  @supports (mix-blend-mode: overlay){
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      top: 0;
+      left: 0;
+      bottom: 0;
+      width: 100%;
+      background-image: url('/images/noise.png');
```
Comment (mdt2) [inline src/scss/_elements.layout.scss]: This background isn't applying to the whole page, it stops at different points depending on screen size. Here, it's stopping at the unsuccessful design system chart:

Screen Shot 2019-07-10 at 10 22 42 AM

File: src/scss/_elements.root.scss [hunk 5]
```diff
@@ -8,3 +8,12 @@
   
   --size: 10rem;
 }
+
+::selection {
```
Comment (mdt2) [inline src/scss/_elements.root.scss]: Never seen something like this. What's it doing?

File: src/scss/_components.diagram.chart.scss (hunk 2 above)
Comment (zastrow) [reply src/scss/_components.diagram.chart.scss]: They’ll only be used here and they’re there for one chart. If there were more instances I would say yes.

File: src/scss/_components.skip-link.scss (hunk 3 above)
Comment (zastrow) [reply src/scss/_components.skip-link.scss]: 🤦‍♂

File: src/scss/_elements.root.scss (hunk 5 above)
Comment (zastrow) [reply src/scss/_elements.root.scss]: Oh, it’s the way you can style the text selection colors when text is highlighted by the cursor.

File: src/scss/_elements.layout.scss (hunk 4 above)
Comment (zastrow) [reply src/scss/_elements.layout.scss]: This is specific to Chrome. Ugh. And the blend-mode looks awful in the browser. Why do people use this browser?!

File: src/scss/_elements.root.scss (hunk 5 above)
Comment (mdt2) [reply src/scss/_elements.root.scss]: Nice! Looks awesome.

**Pull request merged**